cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -465,12 +465,12 @@ func copyNetworkConfig(src, dst string) error {
 		//            have been what was broken so we don't want to break
```

Comment:
it would be good to add comments now about the expectations for each of this copy functions about should src/dst argument point to?  maybe we can also find a name that is clearer than src and dst  srcRoot ? dstRoot ?